readthedocs/core/static-src/core/js/doc-embed/version-compare.js

```diff
@@ -23,7 +23,7 @@ function init(data) {
     warning
       .find('a')
       .attr('href', currentURL)
-      .text(data.version);
+      .text(data.slug);
```

Comment:
This seems to render a different value, right?

What's is `version` and what's `slug` here?

Reply:
Is `version` used in another places? We should check that we can still access it now it's not a tuple anymore.

Reply:
> What's is version and what's slug here?

I think your past self alreay answer that question :smile:  https://github.com/rtfd/readthedocs.org/issues/2357#issuecomment-306661449. Also https://github.com/rtfd/readthedocs.org/issues/2357#issuecomment-307969720

I did a grep and is uses here:

https://github.com/rtfd/readthedocs.org/blob/c5d102a4b119e191fd6a7ea201c776e84dd177f8/readthedocs/core/static-src/core/js/doc-embed/search.js#L16

But again thanks to the implicit cast of js everything was good (at the end everyone was expecting a string here p:)